app/operations/process_payment.rb

```diff
@@ -16,14 +16,14 @@ def initialize(fee_id)
       @fee.update(govpay_payment_status: gp.status,
                   govpay_payment_message: gp.message)
     }
-    if @fee.paid? && !@payment.error? && UPDATE_GLIMR
+    if @fee.paid? && UPDATE_GLIMR
```

Comment:
I think we should separate instantiation from making the govpay call. i.e. ProcessPayment.new(fee_id).call instead of ProcessPayment.new(fee_id)  It just feels a bit icky to have so much stuff happening in the initialize method.